examples/all-clusters-app/all-clusters-common/include/rvc-modes.h

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/mode-base-server/mode-base-server.h>
+#include <app/util/af.h>
+#include <app/util/config.h>
+#include <cstring>
+#include <utility>
+
+template <typename T>
+using List                 = chip::app::DataModel::List<T>;
+using ModeTagStructType    = chip::app::Clusters::detail::Structs::ModeTagStruct::Type;
+using ModeOptionStructType = chip::app::Clusters::detail::Structs::ModeOptionStruct::Type;
```

Comment:
These using clauses are problematic in headers.

Reply:
I removed the use of `using` from all header files [here](https://github.com/project-chip/connectedhomeip/pull/27504/commits/7bb06058b351b4b21e04b70268bc2af4784b56be).